src/Api.Rest/Common/Client/RestClientBase.cs

```diff
@@ -197,6 +197,15 @@ public Task<T> RequestBinary<T>( [NotNull] Func<HttpRequestMessage> requestCreat
 			return PerformRequestAsync( requestCreationHandler, false, BinaryResponseToObjectAsync<T>, true, cancellationToken );
 		}
 
+		/// <summary>
+		/// Start an async operation. Returns an URL to poll for status updates if the operation is accepted, otherwise the operation is done synchronously.
+		/// </summary>
+		/// <exception cref="RestClientException">The response indicated status Accepted, but did not contain polling information.</exception>
+		public Task<Uri> RequestAsyncOperation( [NotNull] Func<HttpRequestMessage> requestCreationHandler, CancellationToken cancellationToken )

```

Comment:
Ich würde hier noch den Return-Wert dokumentieren: Returns a Task that represents the duration of the initial REST request. The result of the task contains the URI for polling the operation result or null in case the server already finshed the request synchronously.

Reply:
Erledigt 👍